### DISCUSSION TITLE
Add list download

### DISCUSSION THREAD

**Discussion by clopolc** [NickvisionApps/Parabolic#741] [Q&A]
Hello,

Thank you for your project. I use "Parabolic" a lot, but I find it lacks an interesting feature: the ability to download a whole series of URLs.
Currently, I have to copy my address, click on "Add", the app offers me the format options, I click on "Download".

Is it possible to provide a list of addresses directly and define the format (minimum resolution, etc.) once to create the downloads?

**Comment by nlogozzo** [comment]
> Is it possible to provide a list of addresses directly and define the format (minimum resolution, etc.) once to create the downloads?

Parabolic already supports downloading playlists with this type of UI.

We could definitely add a feature to add a list of URLs and then treat it as a "playlist" and expose the same UI.

The only thing is we need a good UI mockup of adding multiple URLs.

**Comment by clopolc** [comment]
Thanks for the quick feedback.
Below is a prototype.

;)
![parabolic_mockup](https://github.com/NickvisionApps/Parabolic/assets/160331347/300dfaf5-2e4f-4534-a917-4f5e8708c2fc)

**Comment by nlogozzo** [comment]
Use https://github.com/NickvisionApps/Parabolic/issues/733 to track
